db/seeds.rb

```diff
@@ -121,19 +138,22 @@
                              description: "Ah, we're not surprised you've been lured by this dish. It'll make your tastebuds sing.",
                              price: 9.50,
                              status: "active",
-                             categories: [meat_cat, appetizer_cat])
+                             categories: [meat_cat, appetizer_cat],
+                             image: File.open('public/SirenStew.jpg'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.